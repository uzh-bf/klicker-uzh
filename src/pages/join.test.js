/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Join from './join'

describe('Snapshot-Testing', () => {
  it('Works', () => {
    const component = renderer.create(<Join url={{ query: { shortname: 'rsc' } }} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
