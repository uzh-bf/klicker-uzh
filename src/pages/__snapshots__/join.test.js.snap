// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div>
  <span>
    No session active.
  </span>
</div>
`;
